src/publish.proj

```diff
@@ -3,13 +3,17 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <Import Project="$(ToolsDir)PublishContent.targets" />
   <Import Project="$(ToolsDir)versioning.targets" />
+  <UsingTask TaskName="PushToBlobFeed" AssemblyFile="$(ToolsDir)\Microsoft.DotNet.Build.Tasks.Feed\net46\Microsoft.DotNet.Build.Tasks.Feed.dll" />
```

Comment:
It is not a good idea to hardcode the task path here. I would much perfer we just have `$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.Feed.dll`. We shouldn't need to have this package isolated and we don't want to assume we always load the net46 version of it.